armeria/README.md

```diff
@@ -0,0 +1,36 @@
+# Basic example showing distributed tracing across Armeria apps
```

Comment:
later we'll revisit the README, but this is the same as it was except no maven wrapper